helix-core/src/test/java/org/apache/helix/integration/TestCleanupExternalView.java

```diff
@@ -42,6 +42,7 @@
  * orphan external-view
  */
 public class TestCleanupExternalView extends ZkUnitTestBase {
+  private ExternalView _externalView = null;
```

Comment:
nit: = null is not needed.

Reply:
right. 
just to make sure this is explicit, namely in normal path this is expected to be null all the time.